src/views/app/detail-panel/edit/parts/participants-row.tsx

```diff
@@ -10,9 +10,9 @@ import { Button, ChipInput, Container } from '@zextras/carbonio-design-system';
 import { Controller } from 'react-hook-form';
 import { map, some } from 'lodash';
 import { useTranslation } from 'react-i18next';
-import { ParticipantRole } from '../../../../../types/participant';
 import * as StyledComp from './edit-view-styled-components';
 import { EditViewContext } from './edit-view-context';
+import { ParticipantRole } from '../../../../../commons/utils';
 
 const emailRegex =
```

Comment:
since you have move most of the constants into separate file you can move this regex constant also

Reply:
yes, and probably other constants still scattered around the code. we should create a ticket for it